documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,293 @@
+
+.Operator fails on timeout when calling `provider-validate` webhook
+
+The operator can fail when it calls `provider-validate`. It is advised to change the webhooks timeout to 30 seconds resolves this issue. link:https://issues.redhat.com/browse/MTV-718[(MTV-718)]
+
+.Migrated VM from OpenStack has different power states when source VM is running
+
+A VM migrated from OpenStack does not start automatically when the source VM is running. link:https://issues.redhat.com/browse/MTV-677[(MTV-677)]
+
+.Provider with application credential or user token authentication type remains in `staging` status
+
+When migrating a VM with an application credential or user token authentication type, the migration gets stuck in the `staging` status, without continuing and finishing in the `Ready` status. link:https://issues.redhat.com/browse/MTV-701[(MTV-701)]
+
+.Forklift-controller crashes when creating a plan for a VM with 2 NICs
+
+// What is NIC?
+When creating a {ocp} to {ocp} migration plan for a VM with 2 NICs, it creates a plan with an `Unknown` status. This status causes the `forklift-controller` to fail. link:https://issues.redhat.com/browse/MTV-704[(MTV-704)]
+
+.Difficult to capture errors logged in populator pods from customers
+
+When `ovirt-img` fails to download the image, the reason for the failure does not appear in the populator controller or any events log. The pod keeps restarting after each failure, so customer populator pod error logs can only show the reason for failure if they happen to log exactly when the pod is in an error state. link:(https://issues.redhat.com/browse/MTV-725[(MTV-725)]
+
+.Conversion pod does not use the transfer network
+
+When creating a migration plan using the `transferNetwork`, the `virt-v2v` pod uses the default pod network instead of the transfer network. link:https://issues.redhat.com/browse/MTV-835[(MTV-835)]
+
+.Creating a host secret requires validating the secret before creating the host
+
+When creating a host secret, you must validate the secret before creating the host. The procedure for creating a host secret is as follows:
+
+. Create a secret
+. Create a host
+. Update the host owner reference
+
+link:https://issues.redhat.com/browse/MTV-868[(MTV-868)]
+
+.Migrating VMs from RHV to {ocp-name} Virtualization does not work with an encrypted RBD
+
+Migrating a VM from RHV to {ocp-name} Virtualization fails during `qemu-img create`, returning a `Cannot grow device files` error. The pod sees a smaller file. link:https://issues.redhat.com/browse/MTV-852[(MTV-852)]
+
+.RHV provider `ConnectionTestSucceeded` True response from the wrong URL
+
+The `ConnectionTestSucceeded` condition is set to `True` even when the URL is is different than the API endpoint for the RHV Manager. This issue has been resolved in {project-short} 2.6.0. link:https://issues.redhat.com/browse/MTV-740[(MTV-740)]
```

Comment:
The title should be somehow rewritten to express that this is a Resolved issue. Any great idea? :-)
```suggestion
.RHV provider `ConnectionTestSucceeded` True response from the wrong URL

Previously, the `ConnectionTestSucceeded` condition was set to `True` even when the URL was different than the API endpoint for the RHV Manager. With this update, the issue has been fixed. link:https://issues.redhat.com/browse/MTV-740[(MTV-740)]
```

Reply:
Kept sentence structure as previous versions